src/main/resources/chains/mordor-chain.conf

```diff
@@ -141,6 +141,7 @@
   eth-compatible-storage = true
 
   # Set of initial nodes
+  # https://github.com/etclabscore/mordor/blob/master/static-nodes.json
```

Comment:
The below list doesn't seem to be the same as the from this link :thinking:

Reply:
Not exactly, I only added new nodes, didn't remove ones we had but they didn't. Don't know who is right, I thought taking the union is the conservative approach.

Reply:
Hmm but I couldn't find any of the ones you added on that link :thinking:

Reply:
You're right, I don't know what I mixed up, sorry.

Reply:
Fixed.